modules/seedtagBidAdapter.js

```diff
@@ -1,47 +1,52 @@
 import { isArray, _map, triggerPixel } from '../src/utils.js';
-import { registerBidder } from '../src/adapters/bidderFactory.js'
-import { VIDEO, BANNER } from '../src/mediaTypes.js'
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO, BANNER } from '../src/mediaTypes.js';
 import { config } from '../src/config.js';
 
 const BIDDER_CODE = 'seedtag';
 const SEEDTAG_ALIAS = 'st';
 const SEEDTAG_SSP_ENDPOINT = 'https://s.seedtag.com/c/hb/bid';
 const SEEDTAG_SSP_ONTIMEOUT_ENDPOINT = 'https://s.seedtag.com/se/hb/timeout';
-const ALLOWED_PLACEMENTS = {
-  inImage: true,
-  inScreen: true,
-  inArticle: true,
-  banner: true,
-  video: true
-}
+const ALLOWED_DISPLAY_PLACEMENTS = [
+  'inScreen',
+  'inImage',
+  'inArticle',
+  'inBanner',
+];
```

Comment:
Update available placement for `display` creative (when use `banner` mediaType`)